README.md

```diff
@@ -1,149 +1,86 @@
-# Getting started example for Mbed OS
+# Blinky Mbed OS Example
 
-This guide reviews the steps required to get Blinky with the addition of dynamic OS statistics working on an Mbed OS platform. (Note: To see a rendered example you can import into the Arm Online Compiler, please see our [quick start](https://os.mbed.com/docs/mbed-os/latest/quick-start/online-with-the-online-compiler.html#importing-the-code).)
+## Introduction
```

Comment:
Suggestion: Can we add "Officially supported Application"

Reply:
WHy?

Reply:
As we have so many unofficial application and Evelyne mentioned we need to differentiate from the unsupported application

Reply:
I do not understand the meaning of unofficial application here. What makes one application official and another unofficial?

Reply:
As my knowledge, we going to maintain only 10 Application then how we will show to the user which one is supported by us for any questions/issues.

Reply:
I do not understand the message above.

Reply:
What do you allude by "this"? There are multiple posts in this comment thread.

Reply:
I don't believe it's correct that we're only maintaining 10 applications. However, I could be wrong and would like confirmation. Thanks for asking for clarification. That was confusing of me.